media-libs/svt-av1/svt-av1-0.8.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic cmake
+
+DESCRIPTION="Scalable Video Technology for AV1 (SVT-AV1 Encoder and Decoder)"
+HOMEPAGE="https://github.com/OpenVisualCloud/SVT-AV1"
+
+if [ ${PV} = "9999" ]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/OpenVisualCloud/SVT-AV1.git"
+else
+	SRC_URI="https://github.com/OpenVisualCloud/SVT-AV1/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/SVT-AV1-${PV}"
+fi
+
+LICENSE="BSD-2"
+IUSE="debug"
+SLOT="0"
+
+DEPEND="dev-lang/nasm"
```

Comment:
reading cmakelists.txt upstream, it seems it uses yasm by default

also, this should be in BDEPEND and under abi_x86_... guards because it is only used for x86 (see e.g. x265 ebuild for an example)

Reply:
Seeing as x86 isn't support upstream due to high memory usage, it's probably best to just leave it out. https://github.com/OpenVisualCloud/SVT-AV1/issues/1231#issuecomment-622191344

Reply:
you dropped the dep it seems rather than moving it (it is still needed even with only amd64)

Reply:
side note: abi_x86_... is used for x86, amd64 and x32